src/main/java/com/marafone/marafone/config/WebConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.marafone.marafone.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebConfig {
+    @Bean
+    public WebMvcConfigurer corsConfigurer() {
+        return new WebMvcConfigurer() {
+            @Override
+            public void addCorsMappings(CorsRegistry registry) {
+                registry.addMapping("/**") // Allow all endpoints
+                        .allowedOrigins("http://localhost:5173")
+                        .allowedMethods("GET", "POST", "PUT", "DELETE", "OPTIONS")
+                        .allowedHeaders("*") // allow any headers
```

Comment:
I think this comment is unnecessary, saying that allowedHeaders() allows headers is redundant.

Reply:
I added this comment to emphasize that this line will allow to attach **any** headers.